armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -632,7 +632,17 @@ class DoseResultsMapper(GlobalFluxResultMapper):
     """
     Updates fluence and dpa when time shifts.
 
-    Often called after a depletion step.
+    Often called after a depletion step. It is invoked using :py:meth:`apply() <.DoseResultsMapper.apply>`.
+
+    Parameters
+    ----------
+    depletionSeconds: float, required
+        Length of depletion step in units of seconds
+
+    options: GlobalFluxOptions, required
+        Object describing options used by the global flux solver. A few attributes from
+        this object are used to run the methods in DoseResultsMapper. An example
+        attribute is aclpDoseLimit.
```

Comment:
While I LOVE improving our docs, I'm not sure this is right.

I think this "Parameters" thing only applies to the input arguments of a function/method, right?  And classes don't have parameters.

So, where are these Params coming from?

Reply:
Yay, I'm a big boy now!

I have not seen a consistent approach on this in ARMI, so I should have asked first, sorry. My understanding is that there are different conventions for where to put the arguments to the constructor (`__init__()`) for a class. My previous experience involved putting them in the docstring for the class. Two examples of this would be in `armi.nuclearDataIO.cccc.cccc.IORecord` and `armi.reactor.grids.Grid`. However, I don't think there is a universally-acknowledged Pythonic place to put them, nor probably in ARMI?

Reply:
Hmmm... perhaps that's a fine place to put the `__init__()` constructor documentation.

It just threw me for a loop a bit because I couldn't actually _see_ the constructor for that class.